me.kozec.syncthingtk.yaml

```diff
@@ -41,27 +41,46 @@ modules:
       - /share/man
 
   - name: syncthing
+    disabled: false
     buildsystem: simple
+    build-options:
+        env:
+            GOBIN: /app/bin
+        #build-args:
+        #    - --share=network
+
     build-commands:
       - |
         . /usr/lib/sdk/golang/enable.sh
         export GOPATH=$PWD/go
-        cd go/src/github.com/syncthing/syncthing
-        go run build.go -no-upgrade
-      - install -Dm755 -t /app/bin go/src/github.com/syncthing/syncthing/bin/syncthing
+        go version
+        go mod vendor
+        #go build -mod=vendor
```

Comment:
What is this line?

Reply:
for determining the dependencies to make the generator work.